### PR TITLE
Modified 'icrs_to_spacecraft' to remove 'Nan' vale of 'el'.

### PR DESCRIPTION
**Pull request by sumanbala2210-USRA**
Solved 
"Nan" 'el' Values from 'gdt.core.coords.spacecraft', SpacecraftFrame #41

### CODE REVIEW

**Comment by AdamGoldstein-USRA** [comment]
@sumanbala2210-USRA can you add a unit test [here](https://github.com/sumanbala2210-USRA/gdt-core/blob/5e3c189e0311c5221bd39ed84c6fa78ce8d37819/tests/core/coords/test_spacecraft.py#L95) to confirm that this change fixes the problem?

**Comment by sumanbala2210-USRA** [comment]
Added test case to show 'nan' el value and the update has solved it.

File: tests/core/coords/test_spacecraft.py
Comment (AdamGoldstein-USRA) [inline tests/core/coords/test_spacecraft.py:1]: For future reference, class names should be CamelCase and methods/functions should be snake_case.  But even python standard libraries don't always get this right...

**Review by AdamGoldstein-USRA — APPROVED** [review]
Reviewed code and ran unit test.

**Pull request merged**